src/OCore/OCore.Events/Extensions.cs

```diff
@@ -1,15 +1,18 @@
 ﻿using Orleans;
-using System;
-using System.Collections.Generic;
-using System.Text;
+using System.Threading.Tasks;
 
 namespace OCore.Events
 {
     public static class Extensions
     {
         public static IEventAggregator GetEventAggregator(this IGrainFactory grainFactory)
         {
-            return grainFactory.GetGrain<IEventAggregator>(0);                
+            return grainFactory.GetGrain<IEventAggregator>(0);
+        }
+
+        public static Task RaiseEvent<T>(this IGrainFactory grainFactory, T @event, string streamNameSuffix = null)
+        {
+            return grainFactory.GetEventAggregator().Raise<T>(@event, streamNameSuffix);
```

Comment:
I have added this extension method which I think helps readability to keep grains less noisy.